stable/kommander/values.yaml

```diff
@@ -1,52 +1,3 @@
-kommander-ui:
```

Comment:
this part is replaced with values.yaml in kommander repo: https://github.com/mesosphere/kommander/blob/master/chart/kommander-ui/values.yaml